src/_gettsim/transfers/rente.py

```diff
@@ -625,48 +711,183 @@ def ges_rente_frauen_altersgrenze(
     return out
 
 
-def _ges_rente_arbeitsl_altersgrenze(
-    birthdate_decimal: float,
+def _ges_rente_arbeitsl_altersgrenze_ohne_vertrauensschutz_pruefung(
+    geburtsjahr: int,
+    geburtsmonat: int,
     ges_rente_params: dict,
-    ges_rente_regelaltersgrenze: float,
 ) -> float:
-    """Calculate the age, at which an unemployed is eligible to claim the full
-    pension (without deductions). This pension scheme allows for early retirement
-    with deductions. Hence this threshold is needed as reference for calculating the
-    zugangsfaktor.
+    """Full retirement age for unemployed without Vertrauensschutz.
+
+    Full retirement age depends on birth year and month.
+
+    Parameters
+    ----------
+    geburtsjahr
+        See basic input variable :ref:`geburtsjahr <geburtsjahr>`.
+    geburtsmonat
+        See basic input variable :ref:`geburtsmonat <geburtsmonat>`.
+    ges_rente_params
+        See params documentation
+        :ref:`ges_rente_params <ges_rente_params>`.
+
+    Returns
+    -------
+    Full retirement age for unemployed.
+
+    """
+    if (
+        geburtsjahr
+        <= ges_rente_params["altersgrenze_arbeitsl_abschlagsfrei"][
+            "max_birthyear_old_regime"
+        ]
+    ):
+        out = ges_rente_params["altersgrenze_arbeitsl_abschlagsfrei"][
+            "entry_age_old_regime"
+        ]
+    elif (
+        geburtsjahr
+        >= ges_rente_params["altersgrenze_arbeitsl_abschlagsfrei"][
+            "min_birthyear_new_regime"
+        ]
+    ):
+        out = ges_rente_params["altersgrenze_arbeitsl_abschlagsfrei"][
+            "entry_age_new_regime"
+        ]
+    else:
+        out = ges_rente_params["altersgrenze_arbeitsl_abschlagsfrei"][geburtsjahr][
+            geburtsmonat
+        ]
+
+    return out
+
+
+@dates_active(end="1991-12-31", change_name="_ges_rente_arbeitsl_altersgrenze")
+def _ges_rente_arbeitsl_altersgrenze_ohne_staffelung(
+    ges_rente_params: dict,
+    geburtsjahr: int,  # noqa: ARG001
+) -> float:
+    """Full retirement age for unemployed.
+
+    Before the WFG (Gesetz für Wachstum und Beschäftigung) was implemented in 1997 the
+    full retirement age was the same for every birth cohort.
+
+    ToDo: Remove fake dependency (geburtsjahr).
```

Comment:
```suggestion
    # TODO(@MImmesberger): Remove fake dependency (geburtsjahr).
    # https://github.com/iza-institute-of-labor-economics/gettsim/issues/666
```

Please move this and others outside of docstrings, then it will be flagged by ruff's [flake-8-todo-td](https://docs.astral.sh/ruff/rules/#flake8-todos-td) codes, too, if if does not have this form.

I had converted all to-do comments a while ago. I only realised now that  this did not catch those inside docstrings. Converting all those (some can just be deleted) would be a useful task in the near future. Some are horribly outdated, that is just a distraction.